composer.json

```
@@ -11,7 +11,7 @@
   },
   "require": {
     "cbschuld/browser.php": "^1.9.6",
-    "pimcore/pimcore": "^11.1.4",
+    "pimcore/pimcore": "dev-move-previewthumb-to-adminui as 11.2",
```

```suggestion
    "pimcore/pimcore": "^11.2",
```
Ok, tests seems fine beside `11.x` since the other PR is unmerged

---

Or maybe just need to refactor this instead of turning a protected to public but internal
https://github.com/pimcore/pimcore/pull/16745/commits/d21de326f4a3ddfc28693a1465c2074dbb94162f#diff-c47f0147cc6e364ce6b905a4b8c97bb43ff1bec60048d90d2a062abd165ae4e8R1643-R1646 ?